### ISSUE TITLE
Add more Vietnamese character unicode !

### ISSUE THREAD

**Issue by dinhquochan** [tonsky/FiraCode#185]
I really like this font but my language isn't supported.
Include: á à ả ã ạ ă ắ ằ ẵ ẳ ặ â ấ ầ ẫ ẩ ậ ú ù ũ ủ ụ ư ứ ừ ữ ử ự ó ò õ ỏ ọ ô ố ồ ỗ ổ ộ ơ ớ ờ ỡ ở ợ 

Thank you !

**Comment by tonsky** [comment]
Fira Code is based on Fira Mono, a font developed and published by Mozilla. All characters except ligatures come directly from it. I’m not a font designer, and this is a big job, so I do not plan draw additional alphabets for it. Try suggesting your ideas here: https://github.com/mozilla/Fira. Thx!

**Comment by KytoSai** [comment]
I hope you can support Vietnamese for this font :(

**Comment by dinhquochan** [comment]
@tonsky Fira let's me ask you again after 4 years 😅

**Comment by tonsky** [comment]
I’ll keep it open under Unicode tag, but can’t promise I’ll get to it soon. But if all it takes is implement multiple combining accents, it might be doable

**Comment by Tazshelby2016** [comment]
Yes, it looks like these are latin characters with accents. Several look like they just have acutes and tildes.

**Comment by dinhquochan** [comment]
👍  any updates for this.

**Comment by minhduc0711** [comment]
@dinhquochan sadly no https://github.com/tonsky/FiraCode/issues/1112#issuecomment-675682653
